modernizer-maven-plugin/src/main/java/org/gaul/modernizer_maven_plugin/Modernizer.java

```diff
@@ -162,7 +168,7 @@ public void visit(int version, int access, String name, String signature,
         }
         for (String itr : interfaces) {
             Violation violation = violations.get(itr);
-            checkToken(itr, violation, name, /*lineNumber=*/ -1);
+            checkToken(itr, violation, name, /*lineNumber=*/ -1, "", "");
```

Comment:
Can we add a separate method instead of passing in `""` here? Or at least add `/* methodName=*/` comments as was done previously.